conans/client/new.py

```diff
@@ -111,20 +111,20 @@ class {package_name}TestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     requires = "{name}/{version}@%s/%s" % (username, channel)
     generators = "cmake"
+    cmake = None
 
     def build(self):
-        cmake = CMake(self.settings)
+        self.cmake = CMake(self.settings)
         # Current dir is "test_package/build/<build_id>" and CMakeLists.txt is in "test_package"
-        cmake.configure(self, source_dir=self.conanfile_directory, build_dir="./")
-        cmake.build(self)
+        self.cmake.configure(self, source_dir=self.conanfile_directory, build_dir="./")
+        self.cmake.build(self)
 
     def imports(self):
         self.copy("*.dll", "bin", "bin")
         self.copy("*.dylib", "bin", "bin")
 
     def test(self):
-        os.chdir("bin")
-        self.run(".%sexample" % os.sep)
+        self.cmake.test(self)
```

Comment:
Yes, I got your point. I have transfered the explicit call to cmake file, this is more implicit in conan side.  
I've been thinking about this, use more implicit or explicit statements. It's more like python vs ruby discussion, where python prefer be more explicit.  
My point is, make it easier, but have a good documentation to explain the details.  
If you want to revert this piece, no problem for me.